### DISCUSSION TITLE
[Solved] autogrow column in LayoutMode 'grid-no-grow'

### DISCUSSION THREAD

**Discussion by rutgervanbaren** [KevinVandy/material-react-table#936] [Ideas]
It would be nice if we could select one column to autogrow to fill the remainder of the space in LayoutMode 'grid-no-grow'

**Comment by KevinVandy** [comment]
https://www.material-react-table.com/docs/guides/column-resizing#column-grow

**Comment by rutgervanbaren** [comment]
Nice work mate!
